packages/components/section-header/stories/section-header.stories.tsx

```diff
@@ -25,65 +30,79 @@ export default meta
-        ].map((app) => (
-          <SelectItem
-            key={app.key}
-            value={app.key}
-            rightIcon={
-              <CheckLine
-                className={clsx(
-                  'w-4 h-4 ml-4',
-                  app.key === selectedApp ? 'text-primary-700' : 'invisible',
-                )}
-              />
-            }
-          >
-            {app.name}
-          </SelectItem>
-        ))}
-        <SelectSeparator />
-        <button className="px-2 text-sm font-medium tracking-tight text-primary-700">
-          Create an app
-        </button>
-      </SelectContent>
-    </Select>,
-  ]
+  const actions = (
+    <>
+      <Button color="white" className="w-full sm:w-max">
+        See docs
+      </Button>
+      <Select value={selectedApp} onChange={setSelectedApp}>
+        <SelectTrigger className="rounded bg-neutral-150 shrink-0">
+          <Tooltip content="Selected app" arrow="top-center" className="z-20">
+            <SelectValue placeholder="All apps" />
+          </Tooltip>
+        </SelectTrigger>
+        <SelectContent>
+          {[
+            { key: 'all', name: 'All apps' },
+            { key: '1', name: 'App name 1' },
+            { key: '2', name: 'App name 2' },
+          ].map((app) => (
+            <SelectItem
+              key={app.key}
+              value={app.key}
+              rightIcon={
+                <CheckLine
+                  className={clsx(
+                    'w-4 h-4 ml-4',
+                    app.key === selectedApp ? 'text-primary-700' : 'invisible',
+                  )}
+                />
+              }
+            >
+              {app.name}
+            </SelectItem>
+          ))}
+          <SelectSeparator />
+          <button className="px-2 text-sm font-medium tracking-tight text-primary-700">
+            Create an app
+          </button>
+        </SelectContent>
+      </Select>
+    </>
+  )
 
   return (
     <div className="flex flex-col gap-6">
-      <SectionHeader
-        title="Section Title"
-        description="Lorem ipsum dolor sit amet consectetur. Sed turpis eget sed nullam volutpat integer posuere."
-      />
+      <SectionHeader>
+        <SectionHeaderTitle>
+          Platform Notification
+          <SectionHeaderDescription>
+            Select the platform you want to receive the notification
+          </SectionHeaderDescription>
+        </SectionHeaderTitle>
+      </SectionHeader>
 
-      <SectionHeader
-        title="Section Title"
-        description="Lorem ipsum dolor sit amet consectetur. Sed turpis eget sed nullam volutpat integer posuere."
-        actions={actions}
-      />
+      <SectionHeader>
+        <SectionHeaderTitle>
+          Apps Activity
+          <SectionHeaderDescription>
+            Lorem ipsum dolor sit amet consectetur. Sed turpis eget sed nullam
+            volutpat integer posuere.
+          </SectionHeaderDescription>
+        </SectionHeaderTitle>
```

Comment:
`Description` should not be a child of `Title`. They should live independently inside the parent component as:

```
<SectionHeader>
    <SectionHeaderTitle>...</SectionHeaderTitle>
    <SectionHeaderDescription>...</SectionHeaderDescription>
</SectionHeader>
```